ChaosMod/Util/Graphics.h

```diff
@@ -0,0 +1,20 @@
+
+#include "Natives.h"
+
+namespace Util
+{
+	inline void DRAW_2D_BOX(float x, float y, float width, float height, CRGBA colour, float lineThickness)
```

Comment:
```suggestion
	inline void Draw2DBox(float x, float y, float width, float height, CRGBA colour, float lineThickness)
```
Also CRGBA struct should probably be moved here (and renamed to just RGBA)